modules/tinymce/src/core/main/ts/api/EventTypes.ts

```diff
@@ -173,6 +173,10 @@ export interface OpenNotificationEvent {
   notification: NotificationApi;
 }
 
+export interface DisabledStateChangeEvent {
+  state: boolean;
```

Comment:
Should `state` be `readonly`?

Reply:
It depends the events objects are mutable we don't Object.seal them for example and we do even mutate them sometimes so setting them as readonly would be a bit of a lie. But I guess it's always a lie in type unsafe language like TS.